gef.py

```diff
@@ -8419,7 +8419,7 @@ class HexdumpQwordCommand(HexdumpCommand):
     _example_ = "{:s} qword $rsp L16 REVERSE".format(_cmdline_)
 
     def __init__(self):
-        super(HexdumpCommand, self).__init__(complete=gdb.COMPLETE_LOCATION) #pylint: disable=bad-super-call
+        super().__init__()
```

Comment:
Why did you remove the completion class here?
Also `gdb.COMPLETE_SYMBOLS` is here probably more suited than location

Reply:
the completion is adopted from its super class